client/state/jetpack-settings/modules/test/reducer.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import {
+	JETPACK_MODULE_ACTIVATE,
+	JETPACK_MODULE_ACTIVATE_FAILURE,
+	JETPACK_MODULE_ACTIVATE_SUCCESS
+} from 'state/action-types';
+import {
+	items as itemsReducer,
+	requests as requestsReducer
+} from '../reducer';
+
+import {
+	modules as MODULES_FIXTURE,
+	requests as REQUESTS_FIXTURE
+} from './fixture';
+
+describe( 'items reducer', () => {
```

Comment:
I'd prefer wrapping all reducer tests in a main `describe( 'reducer', () => {` call.